<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试助手-服务端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>537</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="widgetVL" stretch="5,1,1,0,2">
     <item>
      <widget class="QTextEdit" name="showTextEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="middleHL" stretch="2,1,1,2,1,3,2,1,1,2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="midHS3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="protoclLabel">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>通信协议</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="protocolComboBox">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="midHS1">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="ipLabel">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>服务器IP地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ipComboBox">
         <property name="minimumSize">
          <size>
           <width>103</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="midHS2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>端口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portLineEdit">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>8888</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="midHS4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="listenHL">
       <item>
        <widget class="QPushButton" name="startPushButton">
         <property name="text">
          <string>开始监听</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopPushButton">
         <property name="text">
          <string>停止监听</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="interruptPushButton">
         <property name="text">
          <string>断开</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="SocketComboBox" name="socketComboBox"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bottomHL" stretch="10,1">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QTextEdit" name="sendTextEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendPushButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SocketComboBox</class>
   <extends>QComboBox</extends>
   <header>socketComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
